ADAL/automation/tests/interactive/ADALSovereignBlackForestLoginTests.m

```diff
@@ -57,7 +57,8 @@ - (void)testInteractiveAADLogin_withBlackforestUser_withPromptAlways_withLoginHi
                              @"user_identifier" : self.primaryAccount.account,
                              @"user_identifier_type" : @"optional_displayable",
                              @"extra_qp": @"instance_aware=true",
-                             @"authority" : @"https://login.microsoftonline.com/common"
+                             @"authority" : @"https://login.microsoftonline.com/common",
+                             @"resource": @"00000002-0000-0000-c000-000000000000"
```

Comment:
It might require some knowledge to know this is AAD graph. Could we use a variable/constant to replace it in the file? :)